src/js/apps/patients/schedule/reduced-schedule/reduced_schedule_app.js

```diff
@@ -2,18 +2,60 @@ import Radio from 'backbone.radio';
 
 import App from 'js/base/app';
 
+import StateModel from './reduced_schedule_state';
+
+import FiltersApp from 'js/apps/patients/schedule/filters_app';
+
 import SearchComponent from 'js/views/shared/components/list-search';
 
 import { CountView } from 'js/views/patients/shared/list_views';
 
 import { LayoutView, ScheduleTitleView, TableHeaderView, ScheduleListView } from 'js/views/patients/schedule/schedule_views';
 
 export default App.extend({
+  StateModel,
+  childApps: {
+    filters: {
+      AppClass: FiltersApp,
+      regionName: 'filters',
+      restartWithParent: false,
+      getOptions: ['currentClinician'],
+    },
+  },
+  stateEvents: {
+    'change:filters change:states': 'restart',
+  },
+  initListState() {
+    const storedState = this.getState().getStore();
+
+    if (storedState) {
+      this.setState(storedState);
+
+      return;
+    }
+
+    const currentUser = Radio.request('bootstrap', 'currentUser');
+    this.setState({ id: `reduced-schedule_${ currentUser.id }` });
+
+    this.getState().setDefaultFilterStates();
+  },
   onBeforeStop() {
     this.collection = null;
   },
   onBeforeStart() {
-    this.setState({ isReduced: true, searchQuery: '' });
+    if (this.isRestarting()) {
+      const isFiltersSidebarOpen = this.getState('isFiltering');
+
+      if (!isFiltersSidebarOpen) Radio.request('sidebar', 'close');
```

Comment:
I don't *think* this is needed. But I wanted to make sure before I removed it.

My understanding is that this is needed on the worklist and schedule because we want the patient sidebar to close when the app is restarted via the date filter or owner filter. And we want the filters sidebar to not close in that scenario.

But neither of those filters exist on the reduced schedule. So this check shouldn't be necessary?

Reply:
It should account for the only coverage drop from this PR.

Unless I missed something else.

Reply:
yeah so the sidebar close I think is for multi-select and the additional `if (!isFiltersSidebarOpen)` was needed to prevent that happening to the filter sidebar.  Definitely not needed since the other cases is impossible.

Reply:
Okay I just removed that, along with the `state.isFiltering`.

Reply:
Well, we may need the `state.isFiltering`... it's used by the `filters_app`. Let me check that quick.

Reply:
I think it's okay to remove `state.isFiltering` here.

The `filters_app` doesn't use it. It uses a `this.isFiltering` that threw me off ([here](https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/js/apps/patients/schedule/filters_app.js#L27-L32)).